packages/fast-components-react-msft/src/flipper/flipper.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from "react";
+import * as ReactDOM from "react-dom";
+import { get } from "lodash-es";
+import { Button, Foundation, HandledProps, IButtonHandledProps, IButtonUnhandledProps } from "@microsoft/fast-components-react-base";
+import {
+    FlipperDirection,
+    IFlipperHandledProps,
+    IFlipperManagedClasses,
+    IFlipperUnhandledProps
+} from "./flipper.props";
+import { IFlipperClassNameContract, IManagedClasses } from "@microsoft/fast-components-class-name-contracts-msft";
+
+/* tslint:disable-next-line */
+class Flipper extends Foundation<IFlipperHandledProps & IManagedClasses<IFlipperClassNameContract>, React.HTMLAttributes<HTMLButtonElement>, {}> {
+    protected handledProps: HandledProps<IFlipperHandledProps & IManagedClasses<IFlipperClassNameContract>> = {
+        label: void 0,
+        visible: void 0,
+        managedClasses: void 0
+    };
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactElement<HTMLHeadingElement | HTMLParagraphElement> {
+        return (
+            <Button
+                {...this.unhandledProps()}
+                {...this.coerceButtonProps()}
+                managedClasses={this.props.managedClasses}
```

Comment:
Flipper extends button's className contract, so this isn't just Flipper's contract. Removing this will break as will passing `this.props.managedClasses.button`.